app/services/journal_row_converter.rb

```diff
@@ -0,0 +1,24 @@
+class JournalRowConverter
+
+  # Returns an array of generated journal_rows given a single order_detail.
+  def self.from_order_detail(order_detail, journal = nil)
+    [{
+      account: order_detail.product.account,
```

Comment:
Hmm, this is a little weird. I would expect `account: order_detail.account`. I wouldn't change it now, though, as that's what it was before. It actually makes some sense in the context of NU, though. Their "account" is part of the chartstring, which would be something like the numeric code for "Lab Services".